vault/resource_okta_auth_backend.go

```diff
@@ -25,198 +26,204 @@ import (
+			Required:    true,
+			Optional:    false,
+			Description: "The Okta organization. This will be the first part of the url https://XXX.okta.com.",
+		},
 
-			"token": {
-				Type:        schema.TypeString,
-				Required:    false,
-				Optional:    true,
-				Description: "The Okta API token. This is required to query Okta for user group membership. If this is not supplied only locally configured groups will be enabled.",
-				Sensitive:   true,
-			},
+		"token": {
+			Type:        schema.TypeString,
+			Required:    false,
+			Optional:    true,
+			Description: "The Okta API token. This is required to query Okta for user group membership. If this is not supplied only locally configured groups will be enabled.",
+			Sensitive:   true,
+		},
 
-			"base_url": {
-				Type:        schema.TypeString,
-				Required:    false,
-				Optional:    true,
-				Description: "The Okta url. Examples: oktapreview.com, okta.com (default)",
-			},
+		"base_url": {
+			Type:        schema.TypeString,
+			Required:    false,
+			Optional:    true,
+			Description: "The Okta url. Examples: oktapreview.com, okta.com (default)",
+		},
 
-			"bypass_okta_mfa": {
-				Type:        schema.TypeBool,
-				Required:    false,
-				Optional:    true,
-				Description: "When true, requests by Okta for a MFA check will be bypassed. This also disallows certain status checks on the account, such as whether the password is expired.",
-			},
+		"bypass_okta_mfa": {
+			Type:        schema.TypeBool,
+			Required:    false,
+			Optional:    true,
+			Description: "When true, requests by Okta for a MFA check will be bypassed. This also disallows certain status checks on the account, such as whether the password is expired.",
+		},
 
-			"ttl": {
-				Type:         schema.TypeString,
-				Required:     false,
-				Optional:     true,
-				Default:      "0",
-				Description:  "Duration after which authentication will be expired",
-				ValidateFunc: validateOktaTTL,
-				StateFunc:    normalizeOktaTTL,
-			},
+		"ttl": {
+			Type:         schema.TypeString,
+			Required:     false,
+			Optional:     true,
+			Default:      "0",
+			Description:  "Duration after which authentication will be expired",
+			ValidateFunc: validateOktaTTL,
+			StateFunc:    normalizeOktaTTL,
+			Deprecated:   "Deprecated. Please use `token_ttl` instead.",
+		},
 
-			"max_ttl": {
-				Type:         schema.TypeString,
-				Required:     false,
-				Optional:     true,
-				Description:  "Maximum duration after which authentication will be expired",
-				Default:      "0",
-				ValidateFunc: validateOktaTTL,
-				StateFunc:    normalizeOktaTTL,
-			},
+		"max_ttl": {
+			Type:         schema.TypeString,
+			Required:     false,
+			Optional:     true,
+			Description:  "Maximum duration after which authentication will be expired",
+			Default:      "0",
+			ValidateFunc: validateOktaTTL,
+			StateFunc:    normalizeOktaTTL,
+			Deprecated:   "Deprecated. Please use `token_max_ttl` instead.",
```

Comment:
These were previously supported in lieu of the `token_ttl` and `token_max_ttl` fields. Since we are adding parity for those fields, these can now be deprecated.